init/common.js

```diff
@@ -0,0 +1,175 @@
+const config = require("./config");
+const Strapi = require("@strapi/strapi");
+const fs = require("fs");
+const _ = require("lodash");
+
+const setupEnv = (cb) => {
+  fs.writeFile(".env", convertToEnv(config.env), cb);
+};
+
+const convertToEnv = (object) => {
+  let envFile = "";
+  for (const key of Object.keys(object)) {
+    envFile += `${key}=${object[key]}\n`;
+    if (key === 'NODE_ENV' && process.env.NODE_ENV === 'test') {
+        continue;
+    }
+    process.env[key] = object[key];
+  }
+  return envFile;
+};
+
+const setupStrapi = async () => {
+  await Strapi().load();
+  const instance = strapi;
+
+  instance.server.mount();
+  return instance;
+};
+
+const stopStrapi = async (instance) => {
+  await instance.stop();
+};
+
+const cleanUpStrapi = async () => {
+  await Strapi().load();
+
+  const dbSettings = strapi.config.get("database.connection");
+
+  await strapi.destroy();
+
+  if (dbSettings && dbSettings.connection && dbSettings.connection.filename) {
+    const tmpDbFile = dbSettings.connection.filename;
+    if (fs.existsSync(tmpDbFile)) {
```

Comment:
Someone unfamiliar with filesystem might not know what this does. Can you add a comment?

Reply:
Added